src/analysis/convex/specific_functions.lean

```diff
@@ -171,56 +222,144 @@ begin
   fin_cases hm; cc,
 end
 
-lemma convex_on_rpow {p : ℝ} (hp : 1 ≤ p) : convex_on ℝ (Ici 0) (λ x : ℝ, x^p) :=
+-- move this
```

Comment:
Did you forget to move this or do you want to move it later? To where?

Reply:
Oops, fixed!